src/plugins/vis_augmenter/public/vega/helpers.ts

```diff
@@ -322,3 +327,24 @@ export const addPointInTimeEventsLayersToSpec = (
 
   return newSpec;
 };
+
+export const interactionHandlers = {
+  ['visAugmenter.handlePointInTimeClick']: (event: any, item: any) => {
+    if (isPointInTimeView(item)) {
+      alert('clicked annotation');
+    }
+  },
+};
+
+export const addPointInTimeInteractionHandlers = (spec: any) => {
+  spec.interactionHandlers = [
+    {
+      event: 'click',
+      expressionFunction: 'visAugmenter.handlePointInTimeClick',
+    },
+  ];
+};
+
+export const isPointInTimeView = (item: any) => {
+  return item.datum.action === showEventsFlyoutAction;
+};
```

Comment:
not sure if you've played around much with the different scenarios, but what happens if user clicks at a blank space in the chart, or somewhere irrelevant within the canvas? (axes, legend, etc.). I'm assuming `datum` would be empty, or `action` within `datum` would be empty, if clicking on other datapoints.